scratch_script.py

```diff
@@ -0,0 +1,134 @@
+# scratch script to test various implementations in datta-lab-to-nwb
+from datta_lab_to_nwb.markowitz_gillis_nature_2023 import markowitz_gillis_nature_2023behaviorinterface
+from pynwb import NWBFile
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import colorcet as cc
+
+
+def run_conversion():
```

Comment:
Just a reminder from our meeting, the `run_conversion` scratch here can be moved to the `convert_session.py` script with a bare minimal NWB Converter using just this one interface

Reply:
Looks like you're indeed using the converter script at the lower level now; so this function can probably be removed?